examples/misc/test/library_tour/core_test.dart

```diff
@@ -579,8 +579,11 @@ void main() {
       // Specify a date and time in ms since the Unix epoch.
       y2k = DateTime.fromMillisecondsSinceEpoch(946684800000, isUtc: true);
 
-      // Parse an ISO 8601 date.
+      // Parse an ISO 8601 date in the UTC time zone.
       y2k = DateTime.parse('2000-01-01T00:00:00Z');
+
+      // Create a new DateTime from an existing one, adjusting just some properties:
+      var sameTimeLastYear = now.copyWith(year: now.year - 1);
       // #enddocregion DateTime
       assert(2016 < now.year, 'Time travel is verboten!');
       expect(y2k.year, 2000);
```

Comment:
Can you add some sort of test for this new variable? It will also fix the current `unused_local_variable` warning in the dev channel.

```suggestion
      expect(y2k.year, 2000);
      expect(sameTimeLastYear.year, now.year - 1);
```

Reply:
Nice! I wasn't expecting to learn so much from such a "tiny" pr, thanks @parlough